File: Fibonacci/Python/fibonacci.py
# FUNCTION DEFINITION
def fib(n) :
    if n >= 3 :
        return fib(n - 1) + fib(n - 2)
    else :
        return 1

num = input("Enter a number: ")
i_num = int(num)
Fibonacci = fib(i_num) #FUNCTION CALL
print("Fibonacci of",num,"is",Fibonacci)

File: Fibonacci/C++/fibonacci.cpp
#include <iostream>
using namespace std;

int fib(int);

int fib(int n){
    if(n >= 3){
        return fib(n - 1) + fib(n - 2);
    }
    else
    {
        return 1;
    }
}

int main(){
    int num, Fibonacci;
    cout<<"Enter a number: ";
    cin>>num;
    Fibonacci = fib(num);
    cout<<"Fibonacci of "<<num<<" is "<<Fibonacci;
    return 0;
}